cyber/class_loader/shared_library/shared_library.cc

```diff
@@ -43,9 +45,9 @@ void SharedLibrary::Load(const std::string& path, int flags) {
 
   int real_flag = RTLD_LAZY;
   if (flags & SHLIB_LOCAL) {
-    real_flag |= SHLIB_LOCAL;
```

Comment:
Seems like this is the problem. I check the source code and found
```
/* If the following bit is set in the MODE argument to `dlopen',
the symbols of the loaded object and its dependencies are made
visible as if the object were linked directly into the program.  */
#define RTLD_GLOBAL        0x00100
 
/* Unix98 demands the following flag which is the inverse to RTLD_GLOBAL.
The implementation does this by default and so we can define the
value to zero.  */
#define RTLD_LOCAL        0
```
The poco use `RTLD_LOCAL` and `RTLD_GLOBAL`, I may have made a mistake before!